net.imaginaryinfinity.Squiid.json

```diff
@@ -36,7 +36,7 @@
         "sources": [{
                 "type": "archive",
                 "url": "https://gitlab.com/ImaginaryInfinity/squiid-calculator/squiid/-/archive/1.0.3/squiid-1.0.3.tar.gz",
-                "sha256": "c0ba6c1d2d55ef4b9dffc56a2ed887cf1c429d2c6d6a30cec5f5f75bbf587a13"
+                "sha256": "d778e19581339bca9911d6738b3db0cfb92f74259a0b7a4b659d6a5d4375030b"
```

Comment:
1. retagging like that is bad
2. it's `generated-sources.json` that needs to be updated.